RELEASE.md

```diff
@@ -4,16 +4,25 @@ This document describes the steps to release a new version of the crate.
 
 ## Overview
 
-To create a new release, push a new tag following the format `<crate>/v<version>`. 
-This triggers the [release](.github/workflows/release.yml) GitHub Actions workflow.
+To create a new release, either run the release.yml workflow as a workload_dispatch trigger through the GitHub UI, or via the following command substituting the proper values for crate and version.
+```bash
+gh workflow run release.yml -f dry_run=true -f crate=containerd-shim-wasm -f version=0.4.0
+```
 
-In the future we may include a workflow for tagging the release but for now this is manual.
+### Input Values for Release.yml
+- `crate:` [string] the name of the crate within the runwasi project. It should be a directory under `./crates`.
+- `version:` [string] the version of the crate to stamp, tag, and release (e.g., 1.0.0, 0.6.0-rc1)
+- `dry_run:` [boolean] a flag that causes the workflow to run all step except ones that would tag or push artifacts.
 
 The workflow performs the following steps:
+- Verifies inputs
+- Verifies ability to push crates
+- Updates the version of the crate to the version specified in the workflow input
 - Build the crate to be released (determined by the tag), including any artifacts (e.g., associated binaries)
 - Run the tests for that crate (and only that crate!)
+- Publishes to the crates.io
+- Tags the repository for the release
 - Creates a GitHub release for that crate (attaching any artifacts)
-- Publish the crate to crates.io
 
 ### Crate Release Sequence
```

Comment:
I missed this part when I was doing it the first time should be something like 
```suggestion
### Crate Release Sequence

Must release the creates in this order due to depenedencies: 
```

and add `oci-tar-builder` below `containerd-shim-test-modules` but before `containerd-shim-wasm`